src/VisualStudio/Core/Def/Implementation/SemanticClassificationCache/SemanticClassificationCacheIncrementalAnalyzerProvider.cs

```diff
@@ -1,77 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
no more incremental analysis to cause this work to happen.  instead, we just use the pull requests on the OOP server to enqueue work to classify the full doc.